docs/building-a-form/about-the-schema-and-uischema-objects.md

```diff
@@ -230,8 +230,10 @@ The us-forms-system code includes additional `uiSchema` functionality not found
       };
     },
 
-    // Use this when you have an array field that should not be pulled out of the page its
-    // in and shown separately on the review page.
+    // By default, array fields that are displayed on a single page in a form, such as
+    // information for multiple dependents, are displayed as separate pages or sections on
```

Comment:
Really nitpicky, but I might reword this to "are display in a separate section on the review page." Only because the way it works is all the info in the array field is displayed together, and the wording now makes it sound like each item within the array gets its own section, so just wanted to clarify that a bit!

Reply:
Does that mean this:

> To keep the information in a single section on a review page, set this property.

... should be something more like:

"To keep all information together on a review page, set this property."

I'm having a hard time deciphering the difference between when the property is set and not set according to your description.

Reply:
Sorry that wasn't clear. I would keep the wording "To keep information in a single section on a review page, set this property to `true`", just adding "to `true`" at the end. I think that language is the most specific about what's going on. The property only needs to be set to `true` if you want to keep the form fields together in one section; otherwise you don't need to set it all (setting it to `false` is the same as not setting it at all).

Reply:
Got it! Thanks for clarifying.